code/client/src/pages/Create.jsx

```diff
@@ -345,29 +346,22 @@ const Create = () => {
         <Row style={{ marginBottom: 48 }}>
           <Space direction='vertical' size='small'>
             <Hint>Set up a fund recovery address:</Hint>
-            <Select
-              suffixIcon={<SearchOutlined />}
-              placeholder='one1......'
-              style={{ width: isMobile ? '100%' : 500, borderBottom: '1px dashed black' }} bordered={false} showSearch onChange={(v) => setLastResortAddress(v)}
-              value={lastResortAddress}
-              onSearch={(v) => setLastResortAddress(v)}
-            >
-              {Object.keys(wallets).filter(k => wallets[k].network === network).map(k => {
-                const addr = util.safeOneAddress(wallets[k].address)
-                return (
-                  <Select.Option key={k} value={util.safeOneAddress(wallets[k].address)}>
-                    ({wallets[k].name}) {isMobile ? util.ellipsisAddress(addr) : addr}
+            <AddressInput
+              addressValue={lastResortAddress}
+              setAddressCallback={setLastResortAddress}
+              extraSelectOptions={
+                [
+                  <Select.Option key='later' value=''>
+                    <Space size='small' align='baseline'>
+                      I want to do this later in my wallet
+                      <Button type='text' onClick={() => setLastResortAddress({ value: '', label: 'I want to do this later in my wallet' })}>
+                        <CheckOutlined />
```

Comment:
no longer needed, but since we are removing this option altogether - you can update it in the next PR

Reply:
Yes, forgot to remove it, will remove and push.